client/js/app/main/modules/menu/menu.directive.controller.js

```diff
@@ -4,15 +4,38 @@ var angular = require('angular'),
 /*
  * @ngInject
  * */
-module.exports = function ($scope) {
+module.exports = function ($scope, $filter, nowValue, todayService) {
     var vm = this;
     this.isVisible = false;
+    this.tasksList = [];
+    this.badges = [];
+    this.todayLong = $filter('date')(nowValue, "d MMM yyyy").toLowerCase();
 
     $scope.$onRootScope(menuEvents.changeVisibility, function (event, isVisible) {
-        if (angular.isUndefined(isVisible))
+        if (angular.isUndefined(isVisible)) {
             isVisible = !vm.isVisible;
+        }
 
         vm.isVisible = isVisible;
         event.stopPropagation();
     });
-};
\ No newline at end of file
+
+    todayService.getTasks(function (res) {
```

Comment:
Refactor this method, make all steps more clear. Move code to separate methods. For now this method is unreadable.